src/components/Dashboard/LegalObligation.vue

```diff
@@ -1,43 +1,33 @@
-        to date with the Registrar: For example, you must file annual reports, director changes and address changes.
-        <span class="app-blue cursor-pointer" @click="readMoreFlag = true">
-          <span v-if="!readMoreFlag">Read more about your legal obligations...</span>
+      <v-icon color="primary" class="mr-2">mdi-information-outline</v-icon>
+      <div>
+        <span class="font-weight-bold obligations-title">{{getObligations.title}}</span>
+        <span class="obligations-copy">
+          You are required by the <em>{{getObligations.act}}</em> {{getObligations.obligationStatement}}
         </span>
+        <p class="app-blue cursor-pointer mt-2 mb-0" @click="readMoreFlag = true">
+          <span v-if="!readMoreFlag"><u>Read more about your obligations</u></span>
+        </p>
 
         <div v-if="readMoreFlag">
-          <div class="read-more-line">
-            The most common updates you are legally responsible to file include:
-          </div>
-          <div class="read-more-line">
+          <div class="obligations-copy pt-2">{{getObligations.subtitle}}</div>
+          <div class="pt-2">
             <ul>
-              <li>
-                <span class="font-weight-bold">Annual Reports</span> - file an annual report each year within two
-                months after each anniversary of the date on which the company was recognized. This ensures your
-                company remains in good standing with the Registrar.
-              </li>
-              <li>
-                <span class="font-weight-bold">Director changes</span> - update director information within 15 days of
-                any change (appointing, ceasing or updating an existing director's name or address)
-              </li>
-              <li>
-                <span class="font-weight-bold">Company address changes</span> - update any changes to a company's
-                registered or records office addresses as they occur.
+              <li v-for="(changes, index) in getObligations.includedChanges" :key="index" class="obligations-copy">
+                <span class="font-weight-bold">{{changes.label}}</span> - {{changes.description}}
               </li>
             </ul>
           </div>
-          <div class="read-more-line">
-            <span>Find more detailed information <a :href="detailInfoURL" target="_blank">here</a></span>
+          <div class="pt-2">
+            <a :href="getObligations.detailInfoURL" target="_blank">Find out more detailed information</a>
+            <v-icon color="primary" class="ml-1" small>mdi-open-in-new</v-icon>
           </div>
-          <div class="read-more-line">
-            <span class="app-blue cursor-pointer" @click="readMoreFlag = false">Read less...</span>
+          <div class="pt-2">
+            <span class="app-blue cursor-pointer" @click="readMoreFlag = false"><u>Read less</u></span>
```

Comment:
In Create UI, "read more..." and "read less..." are not underlined, but they are rendered as "link buttons". Is there a different design here?

![image](https://user-images.githubusercontent.com/10002889/153954707-b9dfcb84-058f-46bf-9446-3d4be5f77ab3.png)

Reply:
o wow, yea that is interesting.. definitely different design 🤔 
https://projects.invisionapp.com/d/main/#/console/21712216/460798286/preview

These designs show this 'expand' btn with underline, without the `...` etc

![Screen Shot 2022-02-14 at 2 56 48 PM](https://user-images.githubusercontent.com/53542131/153960628-d699691b-0922-4303-b886-6df355c9679f.png)

Reply:
OK, not our problem.